### PR TITLE
new visvis release

### PR DESCRIPTION
**Pull request by almarklein**

### CODE REVIEW

**Comment by almarklein** [comment]
Hey @jakirkham, do you have any idea why CircleCI is not triggered?

**Comment by jakirkham** [comment]
Please see comment ( https://github.com/conda-forge/visvis-feedstock/pull/3#issuecomment-305799684 ).

**Comment by almarklein** [comment]
@jakirkham I tried removing circleCI from my fork, revoking and re-adding. Still no luck. What happens when I merge this? If it is likely that CircleCI *will* kick in post-merge, than I'd be happy to try that; visvis is pure Python and if Travis and Appveyor pass, I wont see how CircleCI wont.

**Comment by almarklein** [comment]
I'm feeling lucky today; I'm just going to press that shiny button and see what happens.

**Comment by jakirkham** [comment]
Also someone had luck deleting their fork and recreating it. Maybe that helps. I'm not entirely sure why issues like this crop up with CircleCI periodically. However they occur with such low frequency that it is hard to investigate.

In any event, would fully expect CircleCI to run on merge as that is not conditional on whatever happened with your fork. Though it would be good if we could get it straightened out before your next release.

xref: https://github.com/conda-forge/staged-recipes/pull/2582#issuecomment-298677239

**Comment by almarklein** [comment]
Oddly, Win and Linux packages are now on anaconda.org, but not OS X, which is build on Travis, right? https://anaconda.org/conda-forge/visvis/files?version=1.10.0

I will try re-creating my clone then.

File: conda-forge.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-appveyor:
-  secure:
-    BINSTAR_TOKEN: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+travis:
+  secure:
+    BINSTAR_TOKEN: TMTlg8cSMJun5aFyOSGTkuYJSvYpxI92qjBABnIzYJOK/tqeiZNlXNWCzaC++QSJtV/jq332YcupCWcN+BfRdyDzy6A7drDQODV1udMM4ZBg1ggOyUDQ06xakXDKQiDSMXSb0ZJaMcMhtS7yi2JISo1dzwGLTw5HhLDY1kgdPrT3UhGp3e/NloD0L+rsGrpAPbcsupARjFjO1nuaA+8stBuyiyC0f/b3RqqAr5qen2gT6r6N6x7fy+WIugWF+vVj3UgXJhZjxKifOKjASHKYVWA7kV4xraMXXj+ZfWl7a98zQZgCJt3CZq+i/t+SV4UGrDSoYRmrS2tf7I1PAi5g6ZDmycWttbfwTRdCZg9PgBFo8M54UxMKo7ZmXM6MTuU1YWazTcFHbpoFKJdSEhhLEBX5BKA9B6nA2bZXZRkFTXipdEufU6hzJ5+9aoU21JIbLDHHUfscdL8tDK0DPkgjCTWe2ARFXAtnTg+tujKP51Fr4cpgJ2VN8uEsn/BZK+LsvngiaIDD5EAWxfnoHRlOzj/1DTbQvgj0vujv98uFXByVobu9vuAicNZbOaFTpYThCKfqdA9ZBl2XO7sGXdoPuIK5FJrZjwH4MQxKfQUMcBgoBP+OgIXkah71DmJge1Br5tTOgTW3Q3Zr9koJ1T6xerdoSajYwjd6CiVbpohB2ms=
```
Comment (isuruf) [inline conda-forge.yml:3]: Why is this value changed? This is why travis-ci didn't upload the conda package

Comment (jakirkham) [reply conda-forge.yml:3]: Thanks @isuruf. Yeah, we need to revert the changes to this file and re-render, @almarklein.

Comment (almarklein) [reply conda-forge.yml:3]: It must have slipped in when I copied stuff over from another feedstock in a desperate attempt to make CircleCI work. Will revert that number, rerender, and make a new PR with a bumped build nr.

Comment (almarklein) [reply conda-forge.yml:3]: Thanks for looking into thus @isuruf !

Comment (almarklein) [reply conda-forge.yml:3]: #5.  Still no CircleCI, unfortunately.

**Pull request merged**